src/item.cpp

```diff
@@ -1246,15 +1269,9 @@ std::string item::info(bool showtext, std::vector<iteminfo> *dump, bool debug) c
                 _("This piece of clothing is very fancy.")));
         }
         if (is_armor() && type->id == "rad_badge") {
-            size_t i;
-            for( i = 1; i < sizeof(rad_dosage_thresholds) / sizeof(rad_dosage_thresholds[0]); i++ ) {
-                if( irridation < rad_dosage_thresholds[i] ) {
-                    break;
-                }
-            }
             dump->push_back(iteminfo("DESCRIPTION",
                 string_format(_("The film strip on the badge is %s."),
-                              _(rad_threshold_colors[i - 1].c_str()))));
+                              _(rad_badge_color(irridation).c_str()))));
```

Comment:
Looks like a double translations (same below), the result of `rad_badge_color` is already translated as the `values` array contains translated entries, doesn't it?

Btw. a C-style array? Why not a C++ array?